### DISCUSSION TITLE
Igx-Combo - itemsHeight?

### DISCUSSION THREAD

**Discussion by mddifilippo89** [IgniteUI/igniteui-angular#12420] [Q&A] [answered]
## Question  
How do you set the height of the igx-combo?

None of this CSS works. Is the displayDensity causing this to not be overriden?

.combo {
  margin: 8px;
  width: 430px;
}

.igx-combo {
  height: 500px;
}

.igx-input-group--box igx-input-group__bundle {
  height: 500px;
}

**Comment by RivaIvanova** [comment, marked as answer]
Hi @mddifilippo89,

I have been looking into your question, however, after observing the provided css I need to ask for some additional information. By “set the height of the igx-combo” do you refer to the height of the IgxCombo dropdown or the height of the IgxCombo input where the values are displayed after selection?

I am asking this question as the last css refers to the IgxCombo input and when a component is using an Emulated ViewEncapsulation this encapsulation should be penetrated using **::ng-deep**.

For example, the following code will penetrate the encapsulation and set the IgxCombo **input** to 100px:

```css
:host::ng-deep {
  .igx-input-group--box .igx-input-group__bundle {
    height: 100px;
  }
}
```

However, setting the height of the IgxCombo dropdown could be achieved by using the [itemsMaxHeight](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/IgxComboComponent.html#itemsMaxHeight) property.

[Here](https://stackblitz.com/edit/angular-zfyvtm-uxnbun?file=src%2Fapp%2Flists%2Fcombo%2Fcombo-styling%2Fcombo-styling.component.scss,src%2Fapp%2Flists%2Fcombo%2Fcombo-styling%2Fcombo-styling.component.html) could be found a small sample demonstrating the abovementioned approaches.
